basics/data-import/pinot-stream-ingestion/import-from-apache-kafka.md

```diff
@@ -77,7 +77,8 @@ You can set the offset to -
 
 * `smallest` to start consumer from the earliest offset
 * `largest` to start consumer from the latest offset
-* `timestamp in milliseconds` to start the consumer from the offset after the timestamp.
+* `timestamp in format yyyy-MM-dd'T'HH:mm:ss.SSSZ` to start the consumer from the offset after the timestamp.
```

Comment:
for period, isn't it an offset criteria of its own? can you add that string here too? it should'nt be part of "timestamp" rt?

Reply:
@npawar If I understand correctly, Right now the text is like
* `smallest` to start consumer from the earliest offset
* `largest` to start consumer from the latest offset
* `timestamp in format yyyy-MM-dd'T'HH:mm:ss.SSSZ` to start the consumer from the offset after the timestamp.
* `datetime duration or period` to start the consumer from the offset after the period eg., '2d'.

Which we wanted to change like the below so that the offset criteria is clearly visible?

* `smallest` to start consumer from the earliest offset
* `largest` to start consumer from the latest offset
* `timestamp`  in format yyyy-MM-dd'T'HH:mm:ss.SSSZ to start the consumer from the offset after the timestamp.
* `period` to start the consumer from the offset after the period eg., '2d'.

Reply:
Ah my bad. Docs look good. I was mistaking it with what's on master. I'm looking into why the docs are not syncing from the PRs. Thanks for this